src/unittest/banded_global_aligner.cpp

```diff
@@ -0,0 +1,1406 @@
+//
+//  banded_global_aligner.cpp
+//  
+//
+//  Created by Jordan Eizenga on 7/26/16.
+//
+//
+
+#include <stdio.h>
+#include "catch.hpp"
+#include "gssw_aligner.hpp"
+#include "vg.hpp"
+#include "banded_global_aligner.hpp"
+
+using namespace google::protobuf;
+
+namespace vg {
+    namespace unittest {
+        
+        TEST_CASE( "Banded global alignment works when read matches exactly",
```

Comment:
I think it's better to avoid "works" as the test name, because it's not that useful an indication of what broke when it fails.